gear-test/spec/test_failed_program_msg_handling.yaml

```diff
@@ -26,6 +26,10 @@ fixtures:
     expected:
       # no active programs in the state
       - programs:
+        - kind: id
+          value: 1
+          terminated: true
```

Comment:
As we've discussed it is ugly and confusing syntax but acceptable for now as a quick and dirty feature.

Reply:
> As we've discussed it is ugly and confusing syntax but acceptable for now as a quick and dirty feature.

So how do you see it in a proper way?

Reply:
> So how do you see it in a proper way?

There are two ways. The simpler is to not flatten `id` and `terminated`:

```yaml
- programs:
  - id:
      kind: number
      value: 1
    terminated: true
```

The more complex is to autodetect the `id` kind:

```yaml
- programs:
  - id: 1 # or `0xd435...da27d` or `5Grwv...utQY` or `alice`
    terminated: true
```

Reply:
I think changing id repr in yamls seems like the next-PR theme

Reply:
> I think changing id repr in yamls seems like the next-PR theme

Totally agree!